CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ Bug fixes:
  * Fixed an issue where `tests/Render.hs` did not gracefully exit in the
    presence of an unknown terminal.
 
+API changes:
+ * Added `Brick.Widgets.Edit.getCursorPosition` helper function.
+
```

Comment:
Just an FYI: please don't add changelog entries. I only add those at release time so I can tell what the appropriate version change should be, and I use the git commit history to help me write the changelog entries for each new release.